### PR TITLE
Alphabetize languages, change weights accordingly

### PR DESCRIPTION
**Pull request by zacharysarah**
This PR:
- Alphabetizes language weighting in `config.toml`
- Adds Korean characters to the Korean language selection

/assign @gochist @ClaudiaJKang

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -131,25 +131,31 @@ description = "Production-Grade Container Orchestration"
...
 description = "Production-Grade Container Orchestration"
 languageName = "Chinese"
 weight = 2
 contentDir = "content/cn"
+
+[languages.ko]
+title = "Kubernetes"
+description = "Production-Grade Container Orchestration"
+# languageName = "Korean"
+languageName = "한국인"
```
Comment (zacharysarah) [inline config.toml]: @gochist @ClaudiaJKang I used Google here, please correct this if it's wrong or if there's a better idiom!

File: config.toml [hunk 2]
```diff
@@ -131,25 +131,30 @@ description = "Production-Grade Container Orchestration"
...
 title = "Kubernetes"
 description = "Production-Grade Container Orchestration"
 languageName = "Chinese"
 weight = 2
 contentDir = "content/cn"
+
+[languages.ko]
+title = "Kubernetes"
+description = "Production-Grade Container Orchestration"
+languageName = "한국인 Korean"
```
Comment (zacharysarah) [inline config.toml]: @gochist @ClaudiaJKang I used Google here, please correct this if it's wrong or if there's a better idiom!

Comment (gochist) [reply config.toml]: Please use 한국어. 한국인 means Korean people.

**Comment by zacharysarah** [comment]
I'm closing this PR to consolidate localization work in #10485, which now includes this PR's commit as a `cherry-pick`. 🍒 

/cc @gochist @ClaudiaJKang

**Pull request closed without merging**